File: class13/cpp/AP1302-Exemplo01.cpp
#include <iostream>
#include <iomanip>
int main() {
    double peso, altura;
    std::cout << "Peso (kg): "; std::cin >> peso;
    std::cout << "Altura (m): "; std::cin >> altura;
    double imc = peso / (altura * altura);
    std::cout << "IMC: " << std::fixed << std::setprecision(1) << imc << std::endl;
    if (imc < 18.5) std::cout << "Classificação: Abaixo do peso" << std::endl;
    else if (imc < 25) std::cout << "Classificação: Peso normal" << std::endl;
    else if (imc < 30) std::cout << "Classificação: Sobrepeso" << std::endl;
    else std::cout << "Classificação: Obesidade" << std::endl;
    return 0;
}

File: class13/python/AP1302-Exemplo01.py
# CalculadoraIMC — Aula 13
# Calculadora de IMC com classificação

peso = float(input("Peso (kg): "))
altura = float(input("Altura (m): "))
imc = peso / (altura ** 2)
print(f"IMC: {imc:.1f}")
if imc < 18.5:
    print("Classificação: Abaixo do peso")
elif imc < 25:
    print("Classificação: Peso normal")
elif imc < 30:
    print("Classificação: Sobrepeso")
else:
    print("Classificação: Obesidade")
